cli/Dockerfile-installed-bionic

```diff
@@ -49,7 +49,7 @@ COPY . /project
 WORKDIR /project/cli
 
 RUN export VERSION=$(../bin/get_version) \
- && sed -i -e s/version.*$/version\ =\ \"${VERSION}\"/ Cargo.toml \
+ && sed -i -e "0,/version.*$/ s/version.*$/version\ =\ \"$(../bin/get_version)\"/" Cargo.toml \
```

Comment:
Why is replacing $VERSION here necessary?

Reply:
The build was failing and this was @rbuysse's fix; he should have more information.